connect-examples/v2/csharp_checkout/Startup.cs

```diff
@@ -44,7 +45,12 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
 
             app.UseStaticFiles();
 
-            app.UseMvc();
+            app.UseRouting();
+            app.UseAuthorization();

```

Comment:
I could be wrong but I don't think this sample app is using the Authorization middleware.